experiments/kompanion/cmd/export/export.go

```diff
@@ -302,6 +308,16 @@ func RunExport(ctx context.Context, opts *ExportOptions) error {
 				Resource: apiResource.Name,
 			}
 
+			if gvr.Group == "" {
```

Comment:
May I ask in which case this will happen? Or is this related to any special treatment on the client-go lib(`clientset.Discovery()`)? I was assuming the kompanion is for ConfigConnector resources, which shall not have non-empty group ( always `*.cnrm.google.com`) nor version (always  `v1alpha1` or `v1beta1`).